backend/prisma/schema.prisma

```diff
@@ -18,3 +18,10 @@ model User {
   salt        String?
   facebookID String?   @unique
 }
+
+model Session {
+  id        String   @id
+  sid       String   @unique
```

Comment:
question for you, do we need to maintain both id and sid? in theory, these could be the same value right? No change needed though cause I think this is fairly common